mobile/lib/features/wallet/wallet_history_item.dart

```diff
@@ -98,10 +98,13 @@ class WalletHistoryItem extends StatelessWidget {
       }
     }();
 
-    var amountFormatter = NumberFormat.compact(locale: "en_IN");
+    var amountFormatter = NumberFormat.compact(locale: "en_UK");
 
     return Card(
       child: ListTile(
+          onTap: () async {
+            await showDialog(context: context, builder: (ctx) => showItemDetails(title, ctx));
```

Comment:
❓What does `showDialog` do and where does it come from? I'm curious because I think our code is just `showItemDetails`, so this must be some kind of pattern.

Reply:
It's a flutter function to show the dialogue as a modal: https://api.flutter.dev/flutter/material/showDialog.html